qiskit/pulse/schedule.py

```diff
@@ -794,36 +794,36 @@ class ScheduleBlock:
     ``ScheduleBlock`` s should be initialized with one of the following alignment contexts:
 
     - :class:`~qiskit.pulse.transforms.AlignLeft`: Align instructions in the
-        `as-soon-as-possible` manner. Instructions are scheduled at the earliest
-        possible time on the channel.
+      `as-soon-as-possible` manner. Instructions are scheduled at the earliest
+      possible time on the channel.
 
     - :class:`~qiskit.pulse.transforms.AlignRight`: Align instructions in the
-        `as-late-as-possible` manner. Instructions are scheduled at the latest
-        possible time on the channel.
+      `as-late-as-possible` manner. Instructions are scheduled at the latest
+      possible time on the channel.
 
     - :class:`~qiskit.pulse.transforms.AlignSequential`: Align instructions sequentially
-        even though they are allocated in different channels.
+      even though they are allocated in different channels.
 
     - :class:`~qiskit.pulse.transforms.AlignEquispaced`: Align instructions with
-        equal interval within a specified duration. Instructions on different channels
-        are aligned sequentially.
+      equal interval within a specified duration. Instructions on different channels
+      are aligned sequentially.
 
     - :class:`~qiskit.pulse.transforms.AlignFunc`: Align instructions with
-        arbitrary position within the given duration. The position is specified by
-        a callback function taking a pulse index ``j`` and returning a
-        fractional coordinate in [0, 1].
+      arbitrary position within the given duration. The position is specified by
+      a callback function taking a pulse index ``j`` and returning a
+      fractional coordinate in [0, 1].
 
     The ``ScheduleBlock`` defaults to the ``AlignLeft`` alignment.
     The timing overlap constraint of instructions is not immediately evaluated,
     and thus we can assign a parameter object to the instruction duration.
     Instructions are implicitly scheduled at optimum time when the program is executed.
 
-    Note that ``ScheduleBlock`` can contain :class:`~qiskit.pulse.instructions.Instruction`s
-    and other ``ScheduleBlock``s to build an experimental program, but ``Schedule`` is not
```

Comment:
Why was the `s` removed here?

Reply:
Because otherwise it would trigger the same issue as the ``pad()`` commit above. https://github.com/Qiskit/qiskit-terra/pull/6288/commits/a6460246fc1759061effb23900dc67568253bd19